### ISSUE TITLE
🐛 Create an InteractiveIcon component to fix accessibility violations

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/platform-ui#1353]
## Describe the bug
We are seeing accessibility violations in all components that use icons because they are labeled with `aria-describedby`.

This was implemented here: https://github.com/icgc-argo/platform-ui/commit/17c7120d069bf4d5b80bf70db8ec6835957d9c36
And is used as a reference to that tooltip custom tooltip component.  @hlminh2000 says we're barely using IDs, and this can be a case-by-case implementation not at the icon level.

Tooltip component is here: https://argo-ui-storybook.netlify.app/?path=/story/uikit-tooltip--basic

## Expected behaviour
Please remove the `aria-describedby` in all icons and standardize this by creating an InteractiveIcon component (within the icon component in storybook). This should combine the tooltip and the icon together. To be used in instances where an icon is used on it's own and requires a tooltip description like:

- [ ] All programs: https://platform-ui.qa.argo.cancercollaboratory.org/submission/program
- [ ] File repo table: https://argo-ui-storybook.netlify.app/?path=/story/components-pages-file-repository-filetable--basic
- [ ] User table:https://platform-ui.qa.argo.cancercollaboratory.org/submission/program/EUCANCAN-BE/manage?activeTab=users
